packages/common/src/messages/notifications.ts

```diff
@@ -0,0 +1,5 @@
+export const listenStreakReminderMessages = {
+  title: 'Keep Your Streak Going!',
+  body: (streak: number) =>
+    `Your ${streak} day listening streak will end in 6 hours! Keep listening to earn daily rewards!`
```

Comment:
any quick way to make the `6` value configurable?

Reply:
I was thinking the same thing! I added it to our meeting notes for tomorrow